src/main/scala/org/allenai/deep_qa/experiments/datasets/Johannes.scala

```diff
@@ -0,0 +1,136 @@
+package org.allenai.deep_qa.experiments.datasets
+
+import org.json4s._
+import org.json4s.JsonDSL._
+
+/**
+ * This object contains a bunch of JValue specifications for Johannes' datasets' data files.
+ */
+object JohannesDatasets {
+
+  def johannesFile(johannesDir: String, split: String, version: String="1.0"): JValue = {
+    val outputDirectory = johannesDir + "processed/"
+    val inputFile = johannesDir + s"v${version}/"+ s"${split}-v${version}.json"
+    val outputFiles = Seq(outputDirectory + s"${split}.tsv")
+    ("sentence producer type" -> "dataset reader") ~
+    ("reader" -> "johannes") ~
+    ("create sentence indices" -> true) ~
+    ("input file" -> inputFile) ~
+    ("output files" -> outputFiles)
+  }
+
+  def johannesDataset(johannesDir: String, split: String, version: String="1.0"): JValue = {
+    val file = johannesFile(johannesDir, split, version)
+    ("data files" -> List(file))
+  }
+
+  def makeMCFile(sentence_params: JValue): JValue = {
```

Comment:
As unfortunate as `Mc` is, I'm starting to lean towards using it instead of `MC`.  I'd recommend changing this.